### DISCUSSION TITLE
Notification template to extract .Entry.Manifest.Labels

### DISCUSSION THREAD

**Discussion by NLZ** [crazy-max/diun#969] [Q&A] [answered]
I would like to get only parts of the `.Entry.Manifest.Labels` in my notification, because as of now it will retruns this:
```
map[org.opencontainers.image.created:2023-09-16T15:30:10.696Z org.opencontainers.image.description:A highly customizable homepage (or startpage / application dashboard) with Docker and service API integrations. org.opencontainers.image.documentation:https://github.com/benphelps/homepage/wiki org.opencontainers.image.licenses:GPL-3.0 org.opencontainers.image.revision:34589f0514990b1bad7a8a252dd260dc164288f1 org.opencontainers.image.source:https://github.com/benphelps/homepage org.opencontainers.image.title:homepage org.opencontainers.image.url:https://github.com/benphelps/homepage org.opencontainers.image.version:v0.6.32]
```

What is the syntax to split this up and get only parts of it, like org.opencontainers.image.version?

**Comment by crazy-max** [comment, marked as answer]
It's using go templates and `Labels` is a map so you can use `{{index .Entry.Manifest.Labels "org.opencontainers.image.version"}}`

More info https://pkg.go.dev/text/template#hdr-Functions

**Reply by NLZ** [reply]
Thank you, thats exactly what I was looking for.
Would it make sense to add this information to https://crazymax.dev/diun/faq/#notification-template ?

**Reply by crazy-max** [reply]
Yes I think we could stipulate go template is being used for templating and add a link to the official docs.
